Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionExtractionWindow.cpp

```diff
@@ -284,6 +332,31 @@ namespace EMStudio
             }
 
             m_warningShowed = false;
+
+            m_rootMotionExtractionWidget->ClearInstances();
+            m_extractRootMotionCheck->setChecked(false);
+            if (numMotions == 1)
```

Comment:
question: Is there anything we should do if `numMotions` isn't `1`?

Reply:
basically hide all ui. We could improve on it by allow batch editing, which could be nice, but need more testing to make sure it can handle that

Reply:
Okay understood, maybe a comment here might be helpful to indicate the intent here? Otherwise looks good 👍